src/applications/search/containers/SearchApp.jsx

```diff
@@ -170,14 +170,14 @@ class SearchApp extends React.Component {
 
     const encodedUrl = encodeURIComponent(url);
     const userAgent = encodeURIComponent(navigator.userAgent);
-    const searchClickTrackingEndpoint = `/search_click_tracking`;
     const encodedQuery = encodeURIComponent(query);
     const apiRequestOptions = {
       method: 'POST',
     };
+    const moduleCode = bestBet ? 'BOOS' : 'I14Y';
 
     apiRequest(
-      `${searchClickTrackingEndpoint}?position=${searchResultPosition}&query=${encodedQuery}&url=${encodedUrl}&user_agent=${userAgent}`,
+      `/search_click_tracking?position=${searchResultPosition}&query=${encodedQuery}&url=${encodedUrl}&user_agent=${userAgent}&module_code=${moduleCode}`,
```

Comment:
removing unnecessary abstraction

Reply:
cross-linking to related issue in Sentry, http://sentry.vfs.va.gov/organizations/vsp/issues/34827/?query=is%3Aunresolved